nextjs-app/src/pages/components/button.tsx

```diff
@@ -0,0 +1,21 @@
+import NavigationLayout from "@/layout/navigation";
+import { PieButton } from "@justeattakeaway/pie-button/dist/react";
+import { PieDivider } from "@justeattakeaway/pie-divider/dist/react";
+import { IconSearch } from "@justeattakeaway/pie-icons-webc/dist/react/IconSearch"
```

Comment:
`@justeattakeaway/pie-icons-webc/react/IconSearch` Should work here. We set up proper paths for the icons library

Reply:
Not the case unfortunately. It looks like we don't actually export that folder in the icons package:

```
  "name": "@justeattakeaway/pie-icons-webc",
  "version": "0.17.2",
  "type": "module",
  "main": "dist/index.js",
  "module": "dist/index.js",
  "files": [
    "dist"
  ],
  ```

Reply:
As discussed, we're currently seeing this issue for Next - https://github.com/vercel/next.js/issues/39375